src/components/OrderForm/index.tsx

```diff
@@ -292,6 +294,14 @@ export class OrderForm extends React.PureComponent<OrderFormProps, OrderFormStat
         );
     }
 
+    private getPercentageButtonCalcValue = (value: number, to: string, type: FormType) => {
+        if (isFinexEnabled() && to.toLowerCase() === 'btc' && type === 'buy' && value === 1) {
```

Comment:
Don't hardcode anything!
Put just the percentage as props of the component.

Reply:
I can put exception settings (markets and coefficients) to constants file and pass as a props to order form. Should it be done this way?